Sources/Routes/Logs.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import Request
+
+public class Logs: Route {
+
+    let userName: String
+    let password: String
+    let authenticationHeader: String
+
+    public init(){
+        self.userName = "admin"
+        self.password = "hunter2"
```

Comment:
We talked earlier today about this being a common way to start but having the significant disadvantage of credentials existing in commit history. Did you consider any alternate ways to manage credentials?

Reply:
I did consider using a .txt file and adding that file to my `.gitignore`, but the major issue I ran into was the fact that cob_spec would fail without having those credentials. Although, it just occurred to me that I could write a script that travis can run with the coded credentials, which can be decoded upon startup. Still, even in that situation someone could easily decode that information from the git history. I'll keep thinking.

Reply:
This isn't a blocker for me. Focus on the other feedback first. Thoughts, @tommcgee?

Reply:
I agree with Craig here, it would be a 'nice to have' as well as a potential way to ramp up your server / get some database experience down the line